packages/api/cms-api/src/dam/files/files.utils.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { svgContainsJavaScript } from "./files.utils";
+
+const cleanSvg = `<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
I'd move the used files into the actual tests for better readability (code is nearer to where it belongs).

And... you may reduce the size of the SVG by removing the path 😁